solutions/day4.py

```diff
@@ -0,0 +1,82 @@
+
+from utilities import input_reader
+
+xmas = "XMAS"
```

Comment:
Typically globally defined constants are written as all caps, i.e. this variable could be called `XMAS` or maybe better `XMAS_STR` or more generally `SEARCH_WORD` or something like that.